modules/client-common/src/main/java/org/apache/ignite/internal/client/proto/ClientColumnTypeConverter.java

```diff
@@ -24,12 +24,12 @@
  */
 public class ClientColumnTypeConverter {
     /**
-     * Converts column type to wire code.
+     * Converts SQL column type to wire code.
      *
      * @param columnType Column type.
      * @return Wire code.
      */
-    public static int columnTypeToOrdinal(ColumnType columnType) {
+    public static int sqlColumnTypeToOrdinal(ColumnType columnType) {
```

Comment:
Why not `ColumnType.ordinal()`? Is this because we want to make sure ordinals do not change in future?

Reply:
Yes, those are client protocol ordinals. But this code is not ideal, we have too many different type codes. I think we even discussed the idea of replacing all type codes with `ColumnType.ordinal` some time ago.

Reply:
Ticket filed to address type code madness: https://issues.apache.org/jira/browse/IGNITE-19270